src/components/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { JSXElementConstructor } from 'react';
+
+import styles from './IconButton.module.scss';
+import { IconProps } from '../Icons/Icons.types';
+
+export interface Properties {
+  className?: string;
+  onClick: () => void;
+
+  Icon: JSXElementConstructor<IconProps>;
+  label?: string;
+  size?: string | number;
+  isFilled?: boolean;
+}
+
+export class IconButton extends React.Component<Properties> {
+  handleClick = (event: React.MouseEvent<HTMLButtonElement>): void => {
+    event.stopPropagation();
+    this.props.onClick();
+  };
+
+  render() {
+    return (
+      <button className={`${styles.IconButton} ${this.props.className}`} onClick={this.handleClick}>
```

Comment:
Wondering if we could reuse the zUI `Button` component in some way here, as it definitely has some shared functionality with `IconButton`.

Reply:
:thinking: I think the duplication is fairly limited. Maybe if the two evolve in similar ways often enough then we can identify what's worth sharing but it seems premature to jump straight to that to me.